diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+diplo - Quick "Diplomacy"
+Without arguments:
+    Print list of civs the player civ has diplomatic relations to.
+    This list might not match the ingame civ list, but is what matters. Does not include 'no contact' civs, etc.
+    Also shows if relations match between player civ and other civ - a mismatch can potentially happen 'naturally', but is important when trying to make peace proper, as both civs need to be at peace with the other.
+With arguments:
+    diplo CIV_ID RELATION
+    Changes the relation to the civ identified by CIV_ID to the one specified in RELATION, making sure that this is mutual.
+    Relations: 0 = Peace; 1 = War; 3 = Alliance (seems to only apply to site govs...?)

```

Comment:
consider allowing the player to specify (case-insensitive) words instead of numbers

Reply:
Should be easy enough to add in form of another function that maps 'war'/'peace' etc to the numbers. I just didn't add it since it feels inconsistent to have that but no recognition of civ names, for which a proper conversion of the names would be needed.

Reply:
Relation argument parsing function is now used for this.